assessments-service/src/main/resources/db/migration/common/V1.43__update_indexes.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE `Assessment` ADD INDEX `assessment_soft_deleted_date` (`softDeletedDate`);
+ALTER TABLE `AssessmentOutcome` ADD INDEX `assessment_outcome_outcome` (`outcome`);
+ALTER TABLE `Assessment` DROP INDEX `assessment_trainee_id`;
+ALTER TABLE `Assessment` ADD INDEX `assessment_duplicates`
+(`traineeId`, `programmeMembershipId`, `reviewDate`, `softDeletedDate`);
+ALTER TABLE `Assessment` ADD INDEX `assessment_gmc_number` (`gmcNumber`);
+ALTER TABLE `Assessment` ADD INDEX `assessment_gdc_number` (`gdcNumber`);
+ALTER TABLE `Assessment` ADD INDEX `assessment_public_health_number` (`publicHealthNumber`);
```

Comment:
I had not considered that the search query is OR'd not AND'd - so a composite index actually would be no use at all. A lesson to me about doing side-work to the main ticket without properly thinking it through...